Bonsai.PulsePal/PulsePal.cs

```diff
@@ -749,7 +749,7 @@ public void WriteProgramHeader(OutputChannel channel, ParameterCode parameter)
 
             readonly int GetVoltageSteps(decimal volts)
             {
-                return (int)(decimal.Ceiling((volts + 10) / 20) * device.dacMaxValue);
+                return (int)(decimal.Ceiling(((volts + 10) / 20) * device.dacMaxValue));
```

Comment:
```suggestion
                return (int)decimal.Ceiling((volts + 10) / 20 * device.dacMaxValue);
```

Because the input is `decimal` and commutativity, we can simplify and remove a pair of parentheses instead of adding one.